### DISCUSSION TITLE
Welcome to DIPY Discussions!

### DISCUSSION THREAD

**Discussion by skoudoro** [dipy/dipy#2306] [3. Community Lounge]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Omarelsherif010** [comment]
Hi everyone, I'm Omar Elsherif. I study **Medical Informatics** at Faculty of Computers & Artificial Intelligence Benha University in Egypt. I want to work in **neuroscience research** using the power of **AI and Data science**. I'm in my **second year at college** now and during the first year, I learned a lot about machine learning, deep learning from Coursera and DataCamp. I also take an advanced nanodegree in Data Analysis from Udacity. I joined **Neuromatch Academy as an interactive student** during the last summer. I want to **join DIPY in GSoC 21 under the umbrellas of INCF**. I most interested in **Image processing and Visualization** so I want to join the project titled (**Extend DIPY Horizon workflow for Visualization**) so any **tips** or something to do in order to start contributing here?
Here is my linkedin profile to keep in touch: https://www.linkedin.com/in/omar-elsherif-data-science/

**Reply by skoudoro** [reply]
Welcome @Omarelsherif010,

Could you transfer you intro to GSoc/GSod Category?  Thank you.

Otherwise we recommend starting by reading all tutorials and run the command-line. Make sure you understand them and if there is any issue during your learning, create a PR to add more information or fix the bug.

**Reply by Omarelsherif010** [reply]
Thanks @skoudoro  for your reply. gonna transfer it now
